### PR TITLE
fix: set-cookie splitter size

### PR DESCRIPTION
**Pull request by abalam666**
## Changes

*please link the issues here*

Changes proposed in this pull request

### CODE REVIEW

File: packages/core/src/utils/cookie.ts [hunk 1]
```diff
@@ -105,6 +105,9 @@ class SecureCookie {
   toJSON() {
     let data: any = {};
     for (let prop in this) {
+      /**
+       * Prefixed fileds with underscore are "server-side-only" protected
```
Comment (loopingz) [inline packages/core/src/utils/cookie.ts:109]: typo on the comment

File: packages/core/src/utils/cookie.ts [hunk 2]
```diff
@@ -120,7 +123,11 @@ class SecureCookie {
         domain: ctx.getHttpContext().getHost(),
         httpOnly: true,
         secure: ctx.getHttpContext().getProtocol() == "https",
-        maxAge: 86400 * 7,
+        /**
+         * Max-Age attribute for a cookie must be seconds,
+         * but this parameter is expected to be mililseconds (eventually received divided by 1000 by the browser)
```
Comment (loopingz) [inline packages/core/src/utils/cookie.ts:128]: another typo ;)

**Comment by loopingz** [comment]
Seems this is in fact fixing the http.ts handler of shell but not lambda.
The maxAge parameter is expressed in seconds in the RFC and on this lib: https://www.npmjs.com/package/cookie

I think this would be more appropriate:
https://github.com/loopingz/webda.io/compare/set-cookie-shell-handler?expand=1

What do you think?

**Comment by abalam666** [comment]
Yes it is definitely better to always follow RFCs , let's do it this way!

**Comment by loopingz** [comment]
Closed as https://github.com/loopingz/webda.io/commit/3e52ca6d06475edd11d31d5cd7d2cfb35f435c24 was merged

**Pull request closed without merging**